README.md

```diff
@@ -2,9 +2,16 @@
 
 This repository contains the scripts and Terraform configurations required to set up a Concourse pipeline for building Bosh releases. Releases each have their own repository and are stored in an S3 bucket when they are built.
 
+
+## Automatic build
+
+All releases that have a [pipeline defined](https://github.com/alphagov/paas-release-ci/blob/master/scripts/build-boshrelease-pipelines.sh) in this repository, as well as releases defined in PRs raised from a branch of this repository, are automatically built by `build concourse`. Releases on master branch get final release builds, whereas releases from PRs get development release build.
+
+The `build concourse` server is deployed in CI account using its own BOSH. You can get information about the server using `make ci showenv` command. This will give you necessary information to log-in to the server. Note that currently the release building pipelines are not public and are only visible after you log in.
```

Comment:
I'd like us to expose the pipelines rather than document the oddity.

There's quite a lot of text here and I don't think it would have helped me when I was confused; having used `alphagov/paas-cf` where all pipelines are exposed and a lack of pipelines means that Concourse hasn't been configured yet.

It looks like the functionality has already been copied in `scripts/deploy-pipeline.sh` so we'd only need to add `export EXPOSE_PIPELINE=1` to `scripts/build-boshrelease-pipelines.sh`.